sofiadinizms/vida-de-inseto
# Feature/level 3

## What I Did
1. Coded the mechanics for the level 3
2. Added temporary assets to simbolize the changes needed on this level

## How to test it
1. Git pull this branch
2. Connect your iPhone to your macbook
3. Run the project on your iPhone
4. Check if an image of Txai and a sun show up
5. Lower your screen brightness to 0
6. Check if there's a "Ganhou!" on your console and the Image of the sun was replaced by a moon
